src/osd/PG.h

```diff
@@ -810,7 +810,14 @@ class PG : public DoutPrefixProvider {
 
   /* You should not use these items without taking their respective queue locks
    * (if they have one) */
-  xlist<PG*>::item stat_queue_item;
```

Comment:
it doesn't look like `stat_queue_item` was used anywhere, i think we can just remove this one